leasing/report/invoice/invoicing_review.py

```diff
@@ -13,11 +13,24 @@
 from enumfields.drf import EnumField
 from rest_framework.response import Response
 
-from leasing.models import ServiceUnit
+from leasing.models import ReceivableType, ServiceUnit
 from leasing.report.excel import FormatType
 from leasing.report.lease.invoicing_disabled_report import INVOICING_DISABLED_REPORT_SQL
 from leasing.report.report_base import ReportBase
-from leasing.report.utils import dictfetchall
+from leasing.report.utils import (
+    calculate_invoice_billing_period_days,
+    dictfetchall,
+    set_active_rent_period,
+)
+
+# Ids of receivable types that are not included in the gaps in the billing periods report
+# These lack the start and end dates of the billing periods by default
+EXCLUDED_RECEIVABLE_TYPE_NAMES = [
```

Comment:
Wonder if there is a better indicator for finding these receivable types than the name, the name could change

Reply:
At this point, I don't know any better criterion for excluding receivable types, or invoices. I looked into the data and saw that there were certain invoices that regularly lacked billing periods, and I collected the receivable types that correlated with such invoices. We could create a ticket for further specifying this logic. We need to figure out which invoices are **meant to be** without billing periods, and which ones should have billing periods.

Reply:
To help keep track of this, should we create a test that checks the fixture to ensure all the exact names exist in the receivable_types?
That would allow someone to notice that their change of removing or renaming a receivable types has side effects for this report

Reply:
Added the test